contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -2320,5 +2320,33 @@ def testSkelBlendShapes( self ) :
 			self.assertAlmostEqual( arm_10["P"].data[i].y, expected_10[i].y, 5 )
 			self.assertAlmostEqual( arm_10["P"].data[i].z, expected_10[i].z, 5 )
 
+	def testRoleBinding( self ) :
+		
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.Point ), "Point" )
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.Vector ), "Vector" )
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.Normal ), "Normal" )
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.UV ), "TextureCoordinate" )
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.Color ), "Color" )
+		self.assertEqual( IECoreUSD.DataAlgo.role( IECore.GeometricData.Interpretation.Rational ), "" )
+
+	def testValueTypeNameBinding( self ) :
+
+		v3 = IECore.V3fData( imath.V3f( 0.0 ) )
+		v2 = IECore.V2fData( imath.V2f( 0.0 ) )
+
+		v3.setInterpretation( IECore.GeometricData.Interpretation.Point )
+		self.assertEqual( IECoreUSD.DataAlgo.valueTypeName( v3 ).type.typeName, "GfVec3f" )
+
+		v3.setInterpretation( IECore.GeometricData.Interpretation.Vector )
+		self.assertEqual( IECoreUSD.DataAlgo.valueTypeName( v3 ).type.typeName, "GfVec3f" )
+
+		v3.setInterpretation( IECore.GeometricData.Interpretation.Normal )
+		self.assertEqual( IECoreUSD.DataAlgo.valueTypeName( v3 ).type.typeName, "GfVec3f" )
+
+		v2.setInterpretation( IECore.GeometricData.Interpretation.UV )
+		self.assertEqual( IECoreUSD.DataAlgo.valueTypeName( v2 ).type.typeName, "GfVec2f" )
+
+		self.assertEqual( IECoreUSD.DataAlgo.valueTypeName( IECore.Color3fData( imath.Color3f( 0.0 ) ) ).type.typeName, "GfVec3f" )
```

Comment:
USDSceneTest is already sprawling, and these are tests for DataAlgo, so could we move them to a new DataAlgoTest file?

Reply:
Addressed in 8e4c777fc41b150dcee66d5775e4261470f666e1